build/gulp/js-bundles.js

```diff
@@ -46,7 +49,13 @@ function processDevBundles(bundles) {
 function muteWebPack() {
 }
 
-gulp.task('js-bundles-prod', ['transpile'], function() {
+gulp.task('version-fix', ['transpile'], function() {
+    return gulp.src(path.join(context.TRANSPILED_PATH, VERSION_FILE_PATH), { base: './' })
+        .pipe(replace(/"\d\d\.\d.*?"/, `"${context.version.script}"`))
```

Comment:
Will this regex /"\d\d\.\d.*?"/ work for the "0.0.0" version placeholder? 

I see this in transpiled scripts and bundles:

    module.exports = "0.0.0";

Reply:
`/[\d.]+/` this should work

Reply:
`/"[\d.]+"/` this should work